src/sql/workbench/parts/notebook/electron-browser/outputs/notebookMarkdown.ts

```diff
@@ -197,7 +199,7 @@ export class NotebookMarkdownRenderer {
 			// ignore
 		}
 		let originIndependentUrl = /^$|^[a-z][a-z0-9+.-]*:|^[?#]/i;
-		if (base && !originIndependentUrl.test(href) && !fs.existsSync(href)) {
+		if (base && !originIndependentUrl.test(href) && (!URI.isUri(href) || !this.existsSync(URI.parse(href)))) {
```

Comment:
this doesn't make it sync, there is no way to make a asynchronous function synchronous